src/Services/GTT/GTT.Api/Configuration/Routes.cs

```diff
@@ -51,5 +51,6 @@ public static class Routes
     /// </summary>
     #region Sports Management
     public const string CreateSport = "v1/sports";
+    public const string GetSports = "v1/sports";
```

Comment:
use same "v1/sports" but change method to GET. 
basically we just need one route v1/sports for
POST Sport
PUT Sport
DELETE Sport
GetSport